marlowe-test/test/Spec/Marlowe/Marlowe.hs

```diff
@@ -151,19 +145,6 @@ tests =
         ]
     ]
 
-maxMarloweValidatorSize :: Int
-#ifdef TRACE_PLUTUS
-maxMarloweValidatorSize = 12737
-#else
-maxMarloweValidatorSize = 12194
-#endif
-
--- | Test that the untyped validator is not too large.
-marloweValidatorSize :: IO ()
```

Comment:
I've made a note to add this test into `marlowe-plutus`, if it is not already there.